internal/kafka/internal/handlers/validation.go

```diff
@@ -351,20 +351,18 @@ func ValidateKafkaClaims(ctx context.Context, validations ...ValidateKafkaClaims
 
 func ValidateKafkaStorageSize(kafkaRequest *dbapi.KafkaRequest, kafkaUpdateReq *private.KafkaUpdateRequest) handlers.Validate {
```

Comment:
can `ValidateKafkaStorageSize` be renamed to `ValidateMaxDataRetentionSize`?

Reply:
thanks, just changed it!